docker-compose.yml

```diff
@@ -25,11 +28,17 @@ services:
             - BEHAVIOR_JSON=client,server,transport
             - BEHAVIOR_THRIFT=client,server,transport
             - BEHAVIOR_HEADERS=client,server,transport,encoding
-            - BEHAVIOR_ERRORS=errors,server
+            - BEHAVIOR_ERRORS=errorshttpout,server
+            # BEHAVIOR_ERRORSHTTPIN TODO
+            - BEHAVIOR_ERRORSTCHOUT=errorstchout,server
```

Comment:
The cases that are testable differ radically between transports. They could be conslidated with a big "if" and the existing modules. We should consider this in a follow-up.